iis-wf/iis-wf-import/src/test/resources/eu/dnetlib/iis/wf/importer/infospace/output/report.json

```diff
@@ -2,6 +2,6 @@
 {"key":"import.infoSpace.dataset", "type":"COUNTER", "value": "2"}
 {"key":"import.infoSpace.docProjectReference", "type":"COUNTER", "value": "1"}
 {"key":"import.infoSpace.project", "type":"COUNTER", "value": "1"}
-{"key":"import.infoSpace.docDedupDocReference", "type":"COUNTER", "value": "1"}
+{"key":"import.infoSpace.docIdentifierMappingDocReference", "type":"COUNTER", "value": "0"}
```

Comment:
I know this is an easy way to make test working again ;) 

But in order to be sure we are still able to import identifier mapping we should update the test input to make sure identifier mapping importer works as it is supposed to work.